README.md

```diff
@@ -173,11 +173,18 @@ This will handle the Eye Tracker (EyeLink set up) and can be called to initializ
 
 Use that to stop your script and only restart when the space bar is pressed.
 
+
+### waitForTrigger
+
+Counts a certain number of triggers coming from the mri/scanner before returning.
+Requires number of triggers to wait for.
+
+
 ## Annexes
 
 ### Experiment template [ WIP ]
 
-### `devSandobox.m` stand-alone
+### `devSandbox.m` stand-alone
```

Comment:
```suggestion
### devSandbox.m stand-alone
```

I was waiting to change this in a repo like this, thanks :)

Reply:
hahaha.

leaving typos around voluntarily to see if we spot them, are you? :stuck_out_tongue:

Reply:
🤪